src/sensor/protocoldetector.cpp

```diff
@@ -130,6 +131,14 @@ bool ProtocolDetector::checkSerial(LinkConfiguration& linkConf)
     }
     qCDebug(PING_PROTOCOL_PROTOCOLDETECTOR) << "Port is open";
     port.setBaudRate(baudrate);
+    port.setBreakEnabled(true);
+    QThread::usleep(500);
+    port.setBreakEnabled(false);
+    QThread::msleep(11);
```

Comment:
Where do these numbers come from? are they the same that Gordon uses?

Reply:
They are the same that gordon uses. these numbers will be documented for the product. This routine should also be compatible with any device.